modules/installation-azure-service-principal.adoc

```diff
@@ -51,6 +48,7 @@ $ az cloud set -n <environment_name>
 ----
 $ az cloud update --profile 2019-03-01-hybrid
 ----
+// Can we update this profile name?
```

Comment:
@MayXuQQ 
Can you suggest a profile name? "2019-03-01-hybrid" sounds very old.

Reply:
I think 2019-03-01-hybrid is the correct value for Azure Stack Hub, it is always a few versions behind the Microsoft hosted Azure clouds (public, gov, etc.)

https://learn.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles?view=azs-2206